pkg/cache/cache.go

```diff
@@ -0,0 +1,89 @@
+//
```

Comment:
Should we put the redis cache behind an interface in case we end up going with something else>

Reply:
good call. Added interface.